Samples/ProvisionDemoLab/azuredeploy.parameters.json

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "newLabName": {
+      "value": "FabrikamDemoLab"
+    },
+    
+    // parameters to set lab policies

```

Comment:
The comments were there before. When I ran it, I didn't have any problems. I think they are fine and JSON doesn't complain about it from what I saw.